README.md

```diff
@@ -59,6 +59,42 @@ For details on the available methods we refer to [iic.jku.at/eda/research/quantu
 
 The simulator is based on the references listed below and can either be used as a **standalone executable** with command-line interface, or as a **library** for the incorporation in other projects.
 
+## Using the Python Bindings / Backend for Qiskit
+
+The backend for Qiskit is available via PyPi. The following code gives an example on the usage:
+
+```python
+from qiskit import *
+import jkq.ddsim
+
+circ = QuantumCircuit(3)
+circ.h(0)
+circ.cx(0, 1)
+circ.cx(0, 2)
+
+print(circ.draw(fold=-1))
+
+provider = jkq.ddsim.JKQProvider()
+
+backend = provider.get_backend('circuit_simulator')
+
+job = execute(circ, backend, shots=10000)
+result = job.result()
+outputstate = result.get_statevector(circ, decimals=3)
+print(outputstate)
+
+counts = result.get_counts(circ)
+print(counts)
+```
+
+The simulator may be used in a more stand-alone fashion:
+
+```python
+...
+sim = jkq.ddsim.pyddsim.CircuitSimulator(circ)
```

Comment:
If you opt to change the `__init__.py` file and the include in this example, this can simply become `sim = ddsim.CircuitSimulator(circ)`